core/lib/spree/preferences/configuration.rb

```diff
@@ -38,18 +38,35 @@ class Configuration
     #     defaults. Set via {#load_defaults}
     attr_reader :loaded_defaults
 
+    attr_reader :load_defaults_called
+
     def initialize
       @loaded_defaults = Spree.solidus_version
+      @load_defaults_called = false
     end
 
     # @param [String] Solidus version from which take defaults when not
     # overriden.
     # @see #load_defaults
     def load_defaults(version)
       @loaded_defaults = version
+      @load_defaults_called = true
       reset
     end
 
+    def check_load_defaults_called(instance_constant_name = nil)
+      return if load_defaults_called || !Spree::Core.has_install_generator_been_run?
+
+      target_name = instance_constant_name || "#{self.class.name}.new"
+      Spree::Deprecation.warn <<~MSG
+        Please, indicate wich Solidus version defaults the application must use.
```

Comment:
Typo: wich → which.

Also, I would reword as follows (for simplicity):

> It's recommended that you explicitly load the default configuration for your current Solidus version. You can do it by adding the following to your Solidus initializer:
>
> ...

Reply:
Done!